validation.go

```diff
@@ -53,7 +73,21 @@ func (v Validator) VerifyRecord(r *pb.Record) error {
 		return ErrInvalidRecordType
 	}
 
-	return val.Func(r.GetKey(), r.GetValue())
+	var author *peer.ID
+	if len(r.GetSignature()) > 0 {
+		pid, err := peer.IDFromString(r.GetAuthor())
+		if err != nil {
+fmt.Printf("Could not parse author to peer ID: %s\n", r.GetAuthor())
```

Comment:
uhm, this seems forgotten from debugging?

Reply:
Oops good catch :)